schemas/Manual.locations.schema.json

```diff
@@ -46,56 +42,59 @@
                     },
                     "uniqueItems": true
                 },
-                "place_item_category": {
-                    "description": "(Optional) Specify the category of item that will be placed at this location. (Ignore Logic)",
+                "region": {
+                    "description": "(Optional) The name of the Region this location is part of.",
```

Comment:
Capitalization question. Should "Region" "Location" and "Item" be capitalized everywhere or nowhere? Inconsistent within this descriptor.

Reply:
I could see equal reasons to have it capitalized and not capitalized. I went with not capitalized for now since it was the most used, but we can always change it whenever. Agree that consistency is the goal either way, so made them consistent.